pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -723,59 +1296,55 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	if err != nil {
 		log.Fatal(err)
 	}
-	getconfigCtx.subVolumeStatus = subVolumeStatus
-	subVolumeStatus.Activate()
+	getconfigCtx.subVolumeStatus.Activate()
```

Comment:
We activate this subscription immediately unlike others, is it expected?

Reply:
My mistake, thanks for finding this.